Catappult/appcoins-wallet-android
# [APPC-2666] Eskills purchase transaction

**What does this PR do?**

Adds eskills purchase transaction type.
This PR should only be reviewed after #181 is merged to dev.

**Database changed?**

No

**Where should the reviewer start?**


**How should this be manually tested?**

Start a e-skills game and check if the iap transaction show as '2048' (with corresponding icon) in the transaction list.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-2666](https://aptoide.atlassian.net/browse/APPC-2666)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 
N/A




**Code Review Checklist**
